src/sys/linux/random.cr

```diff
@@ -30,6 +30,21 @@ module Sys
       end
     end
 
+    # Returns a random UInt32.
+    def self.random_number : UInt32
```

Comment:
`Sys::Random.random_number` is used to feed an integer for `Random#next_u`, not to provide as many bytes for a potential `Random#rand_bytes(type)`. I'll add this, it's great!

Note that I chose `rand_bytes` to mimic the `read_bytes` methods of IO.